server/.env

```diff
@@ -1,3 +1,8 @@
 TEAM_MEMBERS="Aecio, Brian, Sagnik, Ricardo"
 
-JWT_SECRET="secret"
\ No newline at end of file
+JWT_SECRET="secret"
+
+# postgres://<username>:<password>@127.0.0.1:<port, default: 5432>/<database name>
+DEV_DATABASE_URL=postgres://roysagni:password@127.0.0.1:5432/dealsmate_devdb
+TEST_DATABASE_URL=postgres://roysagni:password@127.0.0.1:5432/dealsmate_testdb
+DATABASE_URL=
```

Comment:
The connection will only work if the user has already create that database, right? Can you add some instructions on that? Or you can add code to create the database if it doesn't exist: https://stackoverflow.com/questions/31294562/sequelize-create-database

Reply:
I could add the code, but I'm worried I'll break the server, so for the moment, I've updated DBInstallation.md (not in this pull request yet), with proper instructions.